Sources/protoc-gen-grpc-swift/main.swift

```diff
@@ -156,7 +156,7 @@ func main(args: [String]) throws {
         binaryFile.content = serializedFileDescriptorProto
         response.file.append(binaryFile)
       }
-      if !fileDescriptor.services.isEmpty && (options.generateClient || options.generateServer) {
+      if !fileDescriptor.services.isEmpty && (options.generateClient || options.generateServer || options.generateTestClient) {
```

Comment:
Oh sorry, the formatter doesn't like this, could you run `./scripts/format.sh -f`?

Reply:
Looks like the formatter script isn't happy with `-f`.. I'll put up a separate PR for that.

Reply:
Ah, right, `-f` isn't required but the script tells you to pass it 🙈